content/productivity/range-attack/plugin.js

```diff
@@ -0,0 +1,851 @@
+// author: https://twitter.com/lulks4
```

Comment:
Can we get your pr explanation added into the comments here, thats what people will see when the arrive at the page

Reply:
add commets